devbuild.sh

```diff
@@ -16,7 +16,7 @@ OPTIONS
       (e.g. intel | gnu | cray | gccgfortran)
   -a, --app=APPLICATION
       weather model application to build; for example, ATMAQ for Online-CMAQ
-      (e.g. ATM | ATMAQ | ATMW | S2S | S2SW)
+      (e.g. ATM | ATMAQ | ATMF | ATMW | S2S | S2SW)
```

Comment:
Should this list be pruned to the ones the app supports? 

Do you want to add a blurb about ATMF in the text above the list?

Reply:
Done